devel/lab/Portfile

```diff
@@ -50,3 +50,7 @@ Enter default GitLab token:
 Like hub, lab feels best when aliased as git. In your .bashrc or .bash_profile, put:
 alias git=lab
 "
+
+destroot {
+    xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
+}
```

Comment:
So since this changes the installed files, you need to add the `revision` line and set it to 1.

Reply:
The port did not build, so it's fine to leave `revision` at 0.